packages/web-api/src/WebClient.spec.js

```diff
@@ -1040,25 +1040,22 @@ describe('WebClient', function () {
         });
     });
 
-    it('should have a default conncurrency of 3', function () {
+    it('should have a default conncurrency of 100', function () {
       const client = new WebClient(token);
-      const requests = [
```

Comment:
It seems that this test suite is now broken due to the removal of this local variable. Can you fix it?

```
1) WebClient
       has an option to set request concurrency
         should have a default conncurrency of 100:
     ReferenceError: requests is not defined
      at Context.<anonymous> (src/WebClient.spec.js:1048:26)
      at processImmediate (node:internal/timers:466:21)
```
https://github.com/slackapi/node-slack-sdk/actions/runs/3985639578/jobs/6844647705

Reply:
We squash & merge pull requests, so you're welcome to push small commits to this branch to have the tests re-run.

Reply:
Yup good call, I missed the `request` being utilised, and also incorrectly ordered the for loop.

It should be good now 👍